app_scheduler/AppScheduleEntry entry
---
bool success
string message
